needle/modules/storage/data/keychain_dump.py

```diff
@@ -27,29 +33,72 @@ def __init__(self, params):
     def module_pre(self):
         return BaseModule.module_pre(self, bypass_app=True)
 
+    def save_file(self, remote_name, local_name, silent):
+        """Save it locally"""
+        self.printer.debug("Dumping content of the file: {}".format(remote_name))
+
+        # Pull the file locally
+        outfile = '{}_{}.txt'.format(self.options['output'], str(local_name))
+        self.device.pull(remote_name, outfile)
+
+        # Not silent means that this file should be analyzed
+        if not silent:
+
+            # Apply a filter if specified
+            grep_args = None
+            if self.options['filter']:
+                grep_args = ' -i -a "{}" -C 10 '.format(self.options['filter'])

```

Comment:
I'm curious, why would it leave out content? Just because 10 lines in each direction wouldn't be enough?